Content: Hai Reveal's warlord gets a unique ship (#7435)
endless-sky/endless-sky: data/hai/hai reveal 5 philosophers.txt

			`	"This is General Choot'k, supreme commander of the Unfettered Hai military."`
			`	Marshal Kaine responds, "Good to meet you."`
			`	Choot'k nods but Teeneep continues. "We also need a way to deal with the surface turrets and, short of bringing overwhelming firepower which we do not have, there's only one ship available with the systems to be able to affect that within our timeline. That ship is my Centipede. I have installed electronic warfare systems on it to baffle pirate defenses when I was investigating this before. It is our assessment that I should be able to baffle the turrets long enough to deal with any spaceship defenders and land where the turrets will no longer matter." She looks to Choot'k. "The General has generously offered to loan a drive for my use."`

> 			` Choot'k addresses the group. "Our scan of the facility show one faulty hangar we can open, at the bottom of a kilometer-long tunnel. There is a twenty-meter-thick blast door at the top of the tunnel, made of stronger material than we've ever seen. My flagship has weapons pillaged from another species which can melt through those doors. Then we just fly down the tunnel and pass into the facility.`

Reviewer: Whitespace

> 			`	Choot'k addresses the group. "Our scan of the facility show one faulty hangar we can open, at the bottom of a kilometer-long tunnel. There is a twenty-meter-thick blast door at the top of the tunnel, made of stronger material than we've ever seen. My flagship has enough sheer damage output to destroy the door, combined with the ion to disable its shielding. Then we just fly down the tunnel and pass into the facility.`

Reviewer 2: ```suggestion
			`	Choot'k addresses the group. "Our scan of the facility shows one faulty hangar we can open, at the bottom of a kilometer-long tunnel. There is a twenty-meter-thick blast door at the top of the tunnel, made of stronger material than we've ever seen. My flagship has enough sheer damage output to destroy the door, combined with the ion to disable its shielding. Then we just fly down the tunnel and pass into the facility.`
```

			# This should be adjusted downward if the ship is reused more widely.
			"blast radius" 800
			"hit force" 8000
	outfits
		"Tripulse Shredder" 2
		"Ionic Blaster Prototype" 4
		"Pulse Turret" 4
		"Warder Anti-Missile" 2
		
		"Large Heat Shunt"
		"Fuel Processor"
		"Jump Drive"
		"Quantum Keystone"
		"Systems Core (Large)"
		"Systems Core (Medium)"
		"Wanderer Ramscoop"
		
		"White Sun Reactor"
		"Large Biochemical Cell"
		
		"Afterburner (Comet Class)"
		"Steering (Planetary Class)"
		"Thruster (Planetary Class)"
		
		"Pulse Rifle" 65
		"Fragmentation Grenades" 65
		"Thermal Repeater Rifle" 25
	engine -43 131
		zoom 0.85
	engine 43 131
		zoom 0.85
	engine 0 126.5
		zoom 0.5
	"reverse engine" -61 -18.5
		angle -35
	"reverse engine" 62 -18.5
		angle 35
	"steering engine" -61 -18.5
		zoom 0.6
		angle 145
		left
	"steering engine" 62 -18.5
		zoom 0.6
		angle 215
		right
	"steering engine" -73 46
		zoom 0.5
		angle 80
		left
	"steering engine" 73 46
		zoom 0.5
		angle -80
		right
	"steering engine" -43 131
		zoom 0.45
		right
	"steering engine" 43 131
		zoom 0.45
		left
	gun -42.5 -53.5 "Tripulse Shredder"
	gun 43.5 -54.5 "Tripulse Shredder"

> 	gun -23 -102.5

Reviewer 2: ```suggestion
	gun -23 -102.5 "Ionic Blaster Prototype"
```

For all of these, no?

Author: no need to define all of them they'll just fill slots that are undefined

	gun 24.5 -102.5

> 	gun 12.5 -128

Reviewer 2: ```suggestion
	gun 12.5 -128
```

Though I think perhaps the Tripulses should be moved be moved into these center ports? Unless intent is to have them matched to the port designs??

Author: yeah that was the intent

	gun -10 -128
	turret 0.5 -25.5 "Warder Anti-Missile"

> 	turret -47.5 11.5

Reviewer 2: ```suggestion
	turret -47.5 11.5 "Pulse Turret"
```
Same for these four, yes?

	turret 48.5 11.5
	turret 62.5 76
	turret -61.5 76.5
	turret 0 107.5 "Warder Anti-Missile"
	leak "leak" 35 45
	leak "flame" 60 95
	leak "big leak" 50 45
	explode "tiny explosion" 60
	explode "small explosion" 51
	explode "medium explosion" 54
	explode "large explosion" 53
	explode "huge explosion" 22
	"final explode" "final explosion large"
	description "Before the Shield Beetle, the Hai used the Titan Beetle, a weaker but more capable ship of a similar design. The Emperor Beetle is a Titan coated with many layers of modern armor, making it a more powerful warship than anything else the Hai have built in modern times. Unfortunately, the advanced features are quite expensive, so these ships are used sparingly as flagships and trophy ships."

> 	description "	It was discovered by Unfettered scientists that the cargo bays were more of a vulnerability than anything else in the ship. As it is not meant to plunder but to lead in glorious battle, the cargo bays were removed, allowing for more space for weapons and defensive improvements. This came at the cost of mass and agility, but that is a sacrifice the Unfettered are more than willing to make."

Reviewer 2: ```suggestion
	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo space was removed, converted to more space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```

> 	description "	It was found by Unfettered scientists that the cargo bays were more of a vulnerability than anything else for the ship, as it is not meant to be plundering but rather to lead in glorious battle. Removing those has allowed for more space for weapons and defensive improvements, as well as more flexibility for the ship. This came at the cost of mass and agility, but that is a sacrifice the Unfettered are more than willing to make."

Author: Well flexibility to counter the enemy but you're right that its weird

> 	description "	More than a ship, the Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai or even Unfettered but rather the military leaders of the Unfettered, at the front of those they commands. Some believe it to be foolish to show their leaders' ships in such a manner, but that is not how the Unfettered think. Of course, this ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."

Reviewer 2: I'd rather stick with "no simple" instead of "not just any," but the other changes here are good.

> 	description "	More than a ship, the Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai or even Unfettered but rather the military leaders of the Unfettered, at the front of those they commands. Some believe it to be foolish to show their leaders' ships in such a manner, but that is not how the Unfettered think. Of course, this ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."

Author: ```suggestion
	description "	More than just a ship, the Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai, but the military leaders of the Unfettered leading at the front of those they command. Some believe it to be foolish to show their leaders' ships in such a manner, but that is not how the Unfettered think. Of course, this ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."
```

> 	description "	More than a ship, the Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai or even Unfettered but rather the military leaders of the Unfettered, at the front of those they commands. Some believe it to be foolish to show their leaders' ships in such a manner, but that is not how the Unfettered think. Of course, this ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."

Reviewer 3: Yeah looks good.

			`To complete this mission, you must land with Choot'k, Teeneep, Danforth and Wallace's ships all present in the system and intact...`
	on enter "Devil-Hide"
		conversation

> 			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. In case Choot'k gets into trouble, here's how to recognize his Shield Beetle from a  distance: it is reinforced with a darkened hull and has red engine flares while the others have blue. The baffling won't work forever, so get to it.`

Reviewer 4: General Choot'k's design should be strong enough so he can handle the incoming pirates on his own, without help from escorts, for a few minutes. (Unless you nuke the area.) That's how I tuned the ship design.

			`To complete this mission, you must land with Choot'k, Teeneep, Danforth and Wallace's ships all present in the system and intact...`
	on enter "Devil-Hide"
		conversation

> 			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. In case Choot'k gets into trouble, here's how to recognize his Shield Beetle from a  distance: it is reinforced with a darkened hull and has red engine flares while the others have blue. The baffling won't work forever, so get to it.`

Reviewer: Given the radical difference in sprite, I think you can safelly assume anyone who isn't blind is going to be able to identify the unique ship. It made some sense to mention it before when the sprite itself was the same and only the engine flares and weapons were different, but I don't think it's necessary to describe all the differences now.
Just say "The General's ship is called the <whatever>, make sure it survives." Or similar.

Author: I originally did remove it but Grey didnt agree
Its more distinct now though

Author: Do we even need to say anything? It makes sense the general is on the cool special beetle no?

Reviewer: Well, if the idea is that the General's ship needs to be saved, it perhaps merits some mention, especially if it is acceptable to lose all the other ships. But, no, I don't think you need to describe the ship in detail when it is so obviously unique. At best, just say, "It's the one that looks different."

Author: Its only acceptable to lose the 2 SB

			action
				set "Hai Reveal [B11] Moving Together: label penetration"
			`	Teeneep explains, "We also need a way to deal with the surface turrets and, short of bringing overwhelming firepower which we do not have, there's only one ship available with the systems to be able to affect that within our timeline. That ship is my Centipede. I have installed electronic warfare systems on it to baffle pirate defenses when I was investigating this before. It is our assessment that I should be able to baffle the turrets long enough to deal with any spaceship defenders, and land where the turrets will no longer matter." She looks to Choot'k. "The General has generously offered to loan a drive for my use."`

> 			`	Choot'k addresses the group. "The designs of the facility show a weak point in one of the hangars, which we can exploit. My flagship has advanced prototype weaponry, that, combined with enough raw burst firepower will disable the hangar door without destroying too much of the hangar itself.`

Reviewer 3: ```suggestion
			`	Choot'k addresses the group. "The designs of the facility show a weak point in one of the hangars, which we can exploit. My flagship has advanced prototype weaponry, that combined with enough raw burst firepower will disable the hangar door without destroying too much of the hangar itself.`
```

		zoom 0.6
		angle 215
		right
	"steering engine" -73 46
		zoom 0.5
		angle 80
		left
	"steering engine" 73 46
		zoom 0.5
		angle -80
		right
	"steering engine" -43 131
		zoom 0.45
		right
	"steering engine" 43 131
		zoom 0.45
		left
	gun -42.5 -53.5 "Tripulse Shredder"
	gun 43.5 -54.5 "Tripulse Shredder"
	gun -23 -102.5
	gun 24.5 -102.5
	gun 12.5 -128
	gun -10 -128
	turret 0.5 -25.5 "Warder Anti-Missile"
	turret -47.5 11.5
	turret 48.5 11.5
	turret 62.5 76
	turret -61.5 76.5
	turret 0 107.5 "Warder Anti-Missile"
	leak "leak" 35 45
	leak "flame" 60 95
	leak "big leak" 50 45
	explode "tiny explosion" 60
	explode "small explosion" 51
	explode "medium explosion" 54
	explode "large explosion" 53
	explode "huge explosion" 22
	"final explode" "final explosion large"
	description "Before the Shield Beetle, the Hai used the Titan Beetle, a weaker but more capable ship of a similar design. The Emperor Beetle is a Titan coated with many layers of modern armor, making it a more powerful warship than anything else the Hai have built in modern times. Unfortunately, the advanced features are quite expensive, so these ships are used sparingly as flagships and trophy ships."

> 	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo space was removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."

Reviewer 3: ```suggestion
	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo was removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```

Reviewer 2: Gonna disagree with this one. Removing "cargo" is vastly different from removing "cargo space," and implied distinction between the two only comes afterwards in the sentence.

It is a bit repetitive, but this makes it unclear instead.

Reviewer 5: Maybe swap it for "bay" instead. iirc existing mission texts referred to "cargo bay" a few times.

Author: Feel free to suggest new changes EjoThims I am a bit lost in these proper english arguments x)

Reviewer 2: I like Todd's suggestion above.

Author: Should I just resolve this?

Author: ```suggestion
	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo bays were removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```
like this then right?

Reviewer 3: Looks good.

> 	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo space was removed, converted to more space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."

Reviewer 3: ```suggestion
	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo space was removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```

> 	description "	More than just a ship, the Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai, but the military leaders of the Unfettered leading at the front of those they command. Some believe it to be foolish to show their leaders' ships in such a manner, but that is not how the Unfettered think. Of course, this ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."

Reviewer 3: ```suggestion
	description "	More than just a ship, the Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai, but the military leaders of the Unfettered leading at the front of those they command. Some believe it to be foolish to show their leaders' ships in such a manner, but that is not how the Unfettered think. This ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."
```
